src/app/containers/MostRead/index.amp.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+
+const Amp = () => (
+  <>
+    <Helmet>
```

Comment:
🤔 I'm not clear on why we need this scripts in this container, I thought these headers should be covered in https://github.com/bbc/simorgh/blob/latest/src/server/utilities/constructCspHeader/index.js

Reply:
These scripts are required to use the amp elements: `amp-list`/`amp-mustache`/`amp-bind`. I've only put them here temporarily just to test the implementation will work. But currently, it doesn't work. I will look into after I finish the tests for the index.canonical.jsx